admin/client/App/screens/List/components/ItemsTable/ItemsTableRow.js

```diff
@@ -38,7 +38,7 @@ const ItemsRow = React.createClass({
 		});
 		// item fields
 		var cells = this.props.columns.map((col, i) => {
-			var ColumnType = Columns[col.type] || Columns.__unrecognised__;
+			var ColumnType = Columns[col.type] || Columns.text;
```

Comment:
@JedWatson I had a KS frontend crash due to `Columns.__unrecognised__`, so I'm happy with this line, but I presume that the idea was to actually define a column type `__unrecognised__` so that it would show in a specific way in the UI?